shrimp.js

```diff
@@ -0,0 +1,49 @@
+let shrimp_info = {
+
+    recipeName: "Shrimp Scampi",
+
+    recipeDescription: "You may be asking yourself, what is Shrimp Scampi?\nThis is a common question, because the name can be \nconfusing and scary to new cooks. The word \"scampi\"\n is actually spanish for pasta. So the name of our dish\ntranslates to \"shrimp pasta\" how fun!",
+
+    recipeImage: "blt.jpg",
+
+    ingrediantList: [
+        "10 large shrimp, heads off and unpeeled",
+        "1 tablespoon olive oil",
+        "3 cloves garlic, minced",
+        "4 tablespoons butter",
+        "½ teaspoon dried oregano",
+    ],
+
+    instructionList: [
+        "With a sharp knife, cut shrimp in half so that the meat is exposed in the shell.",
+        "In a small saucepan, heat olive oil and lightly fry the garlic until just softened. Add\n butter and Oregano. Heat until butter has melted.",
+        "Preheat an outdoor grill for medium heat and lightly oil grate.",
+        "Lay shrimp out on a tray and brush generously with the butter mixture. Grill shrimp until lightly golden."
+    ]
+}
+
+let jsonify = () => {
+    let space = document.getElementById("jsondiv");
+    space.innerHTML = "<pre> " + JSON.stringify(shrimp_info, null, 10) + "</pre>";
+
+}
```

Comment:
This function should instead start with an empty json object (ex: let page_json = {}). Then as I explained above you would add key, value pairs to this object (ex: pae_json.ingerdients = [] to make an empty array of ingredients). Then you would want to display this object to the screen.